students/tlugosan/lesson07/test_html_render.py

```diff
@@ -0,0 +1,322 @@
+#!/usr/bin/env python3
+
+import io
+import pytest
+import html_render as hr
+
+
+def render_result(element, cur_ind=''):
+    file_out = io.StringIO()
+    element.render(file_out, cur_ind)
+    return file_out.getvalue()
+
+
+def test_init():
+    e = hr.Element()
+    e = hr.Element("original text")
+
+
+def test_append():
+    e = hr.Element("element to append to")
+    e.append("text to append")
+    file_contents = render_result(e).strip()
+    print(file_contents)
+
+    assert "element to append to" in file_contents
+    assert "text to append" in file_contents
+    assert file_contents.index("element to") < file_contents.index("text to")
+    assert file_contents.startswith("<html>")
+    assert file_contents.endswith("</html>")
```

Comment:
You can assert the full html string instead of just parts of it. Simpler asserts and more coverage.